backend/tests/slack/events/fuzz_tests/user_joind_channel/contribute_test.py

```
@@ -0,0 +1,38 @@
+        mock_active_projects_count,
+        events_enabled,
+        project_count,
+        issue_count,
+    ):
+        settings.SLACK_EVENTS_ENABLED = events_enabled
+        mock_active_projects_count.return_value = project_count
+        mock_open_issues_count.return_value = issue_count
+        mock_slack_event = {"user": "U123456", "channel": OWASP_CONTRIBUTE_CHANNEL_ID}
+        mock_slack_client = MagicMock()
+        mock_slack_client.conversations_open.return_value = {"channel": {"id": "C123456"}}
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)